src/platform/SDL2Window.cxx

```diff
@@ -148,7 +151,10 @@ void SDLWindow::swapBuffers()
     if (! SDL_GL_GetSwapInterval())
         return;
 
-    const int maxRunawayFPS = 65;
+    int maxRunawayFPS = 65;
+    SDL_DisplayMode desktopDisplayMode;
+    if(SDL_GetDesktopDisplayMode(0, &desktopDisplayMode) == 0)
```

Comment:
```suggestion
    if (SDL_GetDesktopDisplayMode(0, &desktopDisplayMode) == 0)
```

Reply:
Agreed. I'll make a mental note to fix this when I do a final interactive rebase to fix and clean up a few things prior to merging.